tensorboard/webapp/app_routing/effects/app_routing_effects.ts

```diff
@@ -88,7 +104,11 @@ export class AppRoutingEffects {
     this.onInit$,
```

Comment:
Might be good to add a comment, either one saying that the 3 source observables in the start of this chain must be appRootLess, or a comment at the top of `userInitNavRoute$`, saying that it must emit data that is appRootLess.